src/mono/mono/mini/method-to-ir.c

```diff
@@ -2238,12 +2238,12 @@ mono_emit_widen_call_res (MonoCompile *cfg, MonoInst *ins, MonoMethodSignature *
 		// LLVM code doesn't uses zero extend the full word while jit expects it.
 		// A proper fix would be to detect if we are actually using llvm code from aot images
 		// or make sure llvm code actually zero extends the return.
-#ifdef MONO_ARCH_LLVM_SUPPORTED
-		gboolean might_use_llvm = TRUE;
+#if defined(MONO_ARCH_LLVM_SUPPORTED) || defined(MONO_ARCH_INTERPRETER_SUPPORTED)
```

Comment:
This change here is just for clarity. `MONO_ARCH_LLVM_SUPPORTED` is already defined everywhere.